src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,26 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<body>` | `.pf-c-login` | `<div>` | Parent container that holds various login components. |
+| `.container-fluid`| `<div>` | `.pf-c-login__container`| `<div>` | Primary container that holds the login component itself. |
+| `.login-pf-page-header` | `<header>` |`.pf-c-login__header` | `<header>` | Header container that sits outside of the login component. |
+| `.login-pf-brand`| `<img>` | `.pf-c-brand` | `<img>` | Patternfly logo brand image tag. |
+| `.login-pf-header` | `<header>` | `.pf-c-login__main-header` | `<header>` | Header container that sits inside of the login component. |
+| `.card-pf` > `<form>` | `<form>` | `.pf-c-login__main` | `<main>` | Main container that holds the form section of the login component. |
```

Comment:
change `.card-pf > <form>` to `.card-pf > form`

Reply:
actually, since there is only 1 `form` on the pf3 login page, you could just include it as `form` - no need for `.card-pf >`

Reply:
- changed ".card-pf > <form>" to just "form".